src/app/fdctl/run/tiles/fd_metric.c

```diff
@@ -263,6 +263,9 @@ prometheus_print( fd_topo_t * topo,
   PRINT( "\n" );
   result = prometheus_print1( topo, out, out_len, "store", FD_METRICS_STORE_TOTAL, FD_METRICS_STORE, PRINT_TILE );
   if( FD_UNLIKELY( result<0 ) ) return result;
+  PRINT( "\n" );
```

Comment:
I have some notion to put `shred` before `store` in the output to preserve the ordering of the tiles in a fragment flow.

Reply:
Yeah, seems reasonable.